docs/uvm_guide.md

```diff
@@ -0,0 +1,99 @@
+# UVM GUIDE For Black Parrot
```

Comment:
Probably makes sense to replace the RAM_IF connection to a UVM connection so that latencies etc can be modelled as well

Reply:
(can't comment on the image above)

Reply:
By this do you mean adding a connection to the UVM testbench similar to the CE interface so that it can be monitored?

Reply:
I added a RAM monitoring option to the UVM testbench.

Reply:
monitor but also driven. Basically rather than a pure verilog model of the memory, supplying the results of the DRAM access (content and latency) should make it easier to increase coverage of test cases

Reply:
That makes sense.  With the feature I added below, if one were to set the RAM interface as active in the test (as a default it has been left as inactive), then it should be able to be driven as well: the code for a driver and sequencer has been committed.